data/gui/window/logging.cfg

```diff
@@ -0,0 +1,192 @@
+#textdomain wesnoth-lib
+###
+### Definition of the window to select logging options.
+### These first five things match the names from logging.cpp
+###
+#define _NAME
+"title_name"
+#enddef
+#define _DEBUG
+"debug"
+#enddef
+#define _INFO
+"info"
+#enddef
+#define _WARN
+"warn"
+#enddef
+#define _ERR
+"err"
+#enddef
+
+#define _GUI_LOGGER_RADIOS GROUP TOOLTIP
+	[column]
+		grow_factor = 0
+
+		border = "all"
+		border_size = 5
+
+		[toggle_button]
+			id = {GROUP}
+			definition = "default"
+			linked_group = {GROUP}
+			tooltip = {TOOLTIP}
+			#wrap = "true"
```

Comment:
Please don't leave commented-out bits in your code unless there is a very good reason for them to be there.